examtool/api/convert.py

```diff
@@ -108,11 +110,29 @@ def consume_rest_of_question(buff):
                     **parse("\n".join(contents)),
                     "options": options,
                     "substitutions": substitutions,
+                    "substitutions_match": substitutions_match
                 }
             else:
                 raise SyntaxError("Unexpected END in QUESTION")
         elif mode == "DEFINE":
-            substitutions[directive] = rest.split(" ")
+            if directive == "MATCH":
```

Comment:
Please extract into a function rather than duplicating logic

Reply:
Yup good point. Was gonna do that but had forgot.